### ISSUE TITLE
[iOS] No Support for Bouncy Castle

### ISSUE THREAD

**Issue by kmohsin11** [Sahamati/Rahasya#32]
Hello, There is no support for bouncy castle on iOS. Apple's CryptoKit just provides method to create Curve25519 key. But there is no way in which we can encode the public key in "ASN1 X9.62" format.
Please help urgently.

**Comment by gsasikumar** [comment]
There is a C test code have you seen that?

**Comment by kmohsin11** [comment]
Yes, checked that. But it does not produce "ASN1 X9.62" format public key. As, "bouncy castle" encodes and decodes using this format I was unable to get result.
Have found the solution though. Used SwiftECC(3rd party) and CryptoKit to fix this.
You may close this now.

**Comment by sooryaraj** [comment]
@kmohsin11 Do you have a sample for that? Am facing same issue
